skbio/test.py

```diff
@@ -0,0 +1,18 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import sys
+from skbio.util import TestRunner
+
+test = TestRunner(__file__).test
+
+if __name__ == '__main__':
+    if test():
+        sys.exit(0)
+    else:
+        sys.exit(1)
```

Comment:
Yeah that's what I started with then based of of [this](http://stackoverflow.com/questions/2764017/is-false-0-and-true-1-in-python-an-implementation-detail-or-is-it-guarante) decided to play it safe. 
```python
sys.exit(0) if test() else sys.exit(1)
``` 
is also an option. Or we could handle it internally in ``test`` and return an exit code rather than a success bool. I do agree the current implementation is a bit clunky.